File: uttam/Java/src/com/main/java/Node.java
package com.main.java;

public class Node {

    public int y;
    public int x;

    public Node(int x, int y) {
        this.x = x;
        this.y = y;
    }

    public int getXCordinate() {
        return x;
    }

    public int getYCordinate() {
        return y;
    }

    @Override
    public String toString() {
        return "(" + x + "," + y + ")";
    }
}

File: NearestNeighbour/Node.scala
package com.rooney.bean

class Node(var name_ : String , var position_ : (Int, Int)) {

	var name : String = name_
	var (x, y) : (Int, Int) = position_

	override def toString() = {
		"name : "+ name +" position,  x: "+x+", y:"+y 
	}

}